FinniversKit/Sources/Components/NMPInfobox/NMPInfoboxView.swift

```diff
@@ -0,0 +1,201 @@
+//
+//  Copyright © FINN.no AS, Inc. All rights reserved.
+//
+
+import SwiftUI
+
+public extension NMPInfoboxView {
+    struct ViewModel: Swift.Identifiable {
+        public struct Interaction {
+            public let title: String
+            public let onTapped: () -> Void
+
+            public init(title: String, onTapped: @escaping () -> Void) {
+                self.title = title
+                self.onTapped = onTapped
+            }
+        }
+
+        public let id = UUID()
+        public let informationType: NMPInfoboxView.InformationType
+        public let title: String
+        public let detail: String
+        public let linkInteraction: Interaction?
+        public let primaryButtonInteraction: Interaction?
+        public let secondaryButtonInteraction: Interaction?
+
+        public init(
+            informationType: NMPInfoboxView.InformationType,
+            title: String,
+            detail: String,
+            linkInteraction: Interaction? = nil,
+            primaryButtonInteraction: Interaction? = nil,
+            secondaryButtonInteraction: Interaction? = nil
+        ) {
+            self.informationType = informationType
+            self.title = title
+            self.detail = detail
+            self.linkInteraction = linkInteraction
+            self.primaryButtonInteraction = primaryButtonInteraction
+            self.secondaryButtonInteraction = secondaryButtonInteraction
+        }
+
+        var titleStyle: Label.Style {
+            .bodyStrong
+        }
+
+        var detailStyle: Label.Style {
+            .body
+        }
+    }
+}
+
+#warning("Og så skal der laves en SwiftUI version som rent faktisk bruger den der Infobox ting")
+public struct NMPInfoboxView: View {
+    public let viewModel: ViewModel
+
+    public init(viewModel: ViewModel) {
+        self.viewModel = viewModel
+    }
+
+    public var body: some View {
+        HStack(alignment: .top, spacing: 8) {
```

Comment:
For spacing, prefer the Fabric (and I assume Warp) spacing constants defined in FinniversKit like `.spacingS`.

Reply:
Ah! Nice catch ✅